tests/docker/Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM python:3.9-slim
 
+ARG GCLOUD_VERSION=338.0.0
+
 RUN apt-get update && apt-get install -y curl \
 	&& curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
 	&& chmod +x ./kubectl \
-	&& mv ./kubectl /usr/local/bin
+	&& mv ./kubectl /usr/local/bin \
```

Comment:
Maybe we could have a different target for `google-cloud-sdk`? It's not needed when running tests locally or with kind in the pipeline, right?

Reply:
I'll create a separate docker file for gitlab as we will probably further divergent dependencies (e.g. awscli etc)